Sources/Sentry/SentryHub.m

```diff
@@ -262,6 +262,42 @@ - (void)captureCrashEvent:(SentryEvent *)event withScope:(SentryScope *)scope
     }
 }
 
+#if SENTRY_HAS_UIKIT
+
+/**
+ * This method expects an abnormal session already stored to disk. For more info checkout: @c
+ * SentryCrashIntegrationSessionHandler
+ */
+- (void)captureFatalAppHangEvent:(SentryEvent *)event
+{
+    // We treat fatal app hang events similar to crashes.
+    event.isCrashEvent = YES;
+
+    SentryClient *client = _client;
```

Comment:
`l`: should we be more explicit with nullability in general and add all `Nonnull` or `Nullable` keywords?